### ISSUE TITLE
Make detectPackage from '@polkadot/util more inclusive 🙏

### ISSUE THREAD

**Issue by yangwao** [polkadot-js/common#521]
We would like to keep using all bells and whistles within the keyring ecosystem where great features are made... but there is one thing we've found. 🤔

We are updating 📦 @vue-polkadot/vue-keyring and we've found out 
```import { detectPackage } from '@polkadot/util';```
is not very inclusive because of checking specific package name family. 
Previous `assertSingletonPackage` worked well 👌😌

Idea is make PR to allow @vue-polkadot as well or check from package.name.

https://github.com/polkadot-js/common/blob/3112e43715cc9b5adbb0e879830ad76fd21bffab/packages/util/src/detectPackage.ts#L59

Error we got and being stuck. Previous assert
![image](https://user-images.githubusercontent.com/5887929/74332190-82507c00-4d95-11ea-8d0f-94e6f5bcdba3.png)

**Comment by jacogr** [comment]
I would rather suggest not using detectPackage if it creates issues for your setup. It sadly is the single util that is more-or-less tied to the namespace.

The previous assert* version had some serious shortcomings which, when actually used, gave limited info and was actually mis-used (finger troubles) due to the lack of checks. So the detect* version is a bit more stringent to weed out some issues in the `@polkadot/` namespace and tries to be more informative when issues do occur. (Currently everything is also not yet where they should be, but moving in the right direction - more ideas on improvements than actual time.)

I didn't take explicit forks in account (it was just never a requirement), since I generally assume that most forks would contribute back upstream (Just out of my experience in the past dealing with various teams building out.)

TL;DR Would just suggest skipping it in this case.
